checks.d/vsphere.py

```diff
@@ -602,6 +607,18 @@ def _cache_morlist_raw_atomic(self, i_key, obj_type, obj, tags, regexes=None):
                 if not match:
                     self.log.debug(u"Filtered out VM {0} because of vm_include_only_regex".format(obj.name))
                     return
+            # Also, if include_only_marked is true, then check if there exists a
+            # custom field with the value DatadogMonitored
+            if _is_affirmative(include_only_marked):
```

Comment:
nitpick, but feel free to ignore: I'd probably put `is_affirmative()` here:

`include_only_marked = _is_affrmative(instance.get('include_only_marked', False))` on line 653